thirdeye/thirdeye-frontend/app/helpers/manage-alert-utils.js

```diff
@@ -182,7 +234,7 @@ export function buildAnomalyStats(alertEvalMetrics, mode) {
       title: 'Number of anomalies',
       key: 'totalAlerts',
       text: 'Estimated average number of anomalies',
-      tooltip
+      tooltip,
```

Comment:
nit: trailing commas in objects are not necessary and our best practices discourage it.

Reply:
oops - didn't see that